CHANGES.rst

```diff
@@ -75,6 +75,8 @@ astropy.samp
 astropy.stats
 ^^^^^^^^^^^^^
 
+- Improve performance of `bayesian_blocks()` by removing one `no.log()` call
```

Comment:
Not sure if this belongs in "new feature," but this isn't a bug fix or API change either, so maybe under whatever we have for "others"?

```suggestion
- Improve performance of ``bayesian_blocks()`` by removing one ``no.log()`` call. [#11356]
```

Reply:
```suggestion
- Improve performance of `bayesian_blocks()` by removing one `np.log()` call
```